src/datasources/__tests__/tool.test.js

```diff
@@ -331,3 +331,96 @@ describe('tool', () => {
     ).rejects.toThrow(Error)
   })
 })
+
+describe('subscription#delete', () => {
+  const OLD_ENV = process.env
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+
+    jest.restoreAllMocks()
+
+    process.env = { ...OLD_ENV }
+
+    process.env.cmrRootUrl = 'http://example.com'
```

Comment:
I've been trying to be more specific in the URLs in tests to indicate what service its calling, `example-cmr` in this case -- it makes the testing a bit more specific and easier to read.